### DISCUSSION TITLE
World Package 1.5.1버전이 업데이트 되었습니다

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#994] [1. Announcements]
안녕하세요, 제페토 월드 크리에이터 여러분
World Package 1.5.0 버전에서 이슈가 발견되어 1.5.1 버전 핫픽스를 진행하게 되었습니다 🙏
버전 업 후 컴파일러 재실행을 위해 유니티를 재시작 해 주세요.

### 안내
[패키지 업데이트 방법](https://studio.zepeto.me/kr/guides/updating-package)
[1.5.0 릴리즈 노트 보기](https://studio.zepeto.me/kr/guides/world-150)
[1.5.1 릴리즈 노트 보기](https://studio.zepeto.me/kr/guides/world-151)

### 개선된 기능

- 1.5.0 업데이트 내역 중 GameObject 타입 변수의 null 비교 가능하도록 수정한 부분에서 컴파일 오류가 발생하여 롤백하였습니다.

감사합니다.

**Comment by koldshine** [comment]
안녕하세요
GameObject.FindObjectOfType에서 includeInactive가 적용이 안되는 문제도 수정된건가요?
